build.sbt

```diff
@@ -3,9 +3,11 @@ import sbt.Def
 import scoverage.ScoverageKeys
 import uk.gov.hmrc.DefaultBuildSettings
 
+resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
+
```

Comment:
This same Maven repository is also declared in project/plugins.sbt so do we need it here too?  I think most of the other services only define resolvers in one place.